k8s.io/configmap-nginx.yaml

```diff
@@ -258,7 +258,6 @@ data:
         location / {
           rewrite ^/bounty$          https://github.com/kubernetes/kubernetes.github.io/issues?q=is%3Aopen+is%3Aissue+label%3ABounty redirect;
           rewrite ^/bot-commands$    https://prow.k8s.io/command-help.html redirect;
-          rewrite ^/cluster-api$     https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md redirect;
```

Comment:
this only affects https://go.k8s.io/cluster-api, so it won't fix the original issue.

the section to fix so that https://sigs.k8s.io/cluster-api/ does the right thing would be 
https://github.com/kubernetes/k8s.io/blob/04adf1d3ff64805e23dafbf80898b5c60d050b91/k8s.io/configmap-nginx.yaml#L80-L111

Reply:
it'd also be good to extend test.py with a test case for this.

Reply:
Is this PR intending to both fix the go-get and the go.k8s.io URL?  In this case "go" doesn't mean "golang", just "go to".  I don't think we want this part - it's sort of a breaking "API" change.